flake.lock

```diff
@@ -681,7 +681,7 @@
     "solc-macos-amd64-list-json": {
       "flake": false,
       "locked": {
-        "narHash": "sha256-akcOahJE/QR0jCPvo4aBf0loJt9P8zvP2E7oCBCRIho=",
+        "narHash": "sha256-f5kzXsfnxgad+1FxaGOlflIUVt8RPNBGS5wkqb63DVA=",
```

Comment:
What made this change necessary?

Reply:
In previous runs, and on other PRs ([example](https://github.com/runtimeverification/kontrol/actions/runs/9107939716/job/25037927689?pr=567#step:5:705)), that hash was not matching anymore. I tried running `nix --extra-experimental-features nix-command --extra-experimental-features flakes flake update` but without any resulting changes. This was my next attempt.